asdf_astropy/converters/slicedwcs/slicedwcs.py

```diff
@@ -0,0 +1,34 @@
+from asdf.extension import Converter
```

Comment:
I believe the general layout for this package is to put converters in submodules similar to the layout in astropy. Would you move this and `fitswcs.py` (and the corresponding tests) into a `wcs` submodule? Having them both contained in the new `wcs` submodule is sufficient (so the `SlicedWCSConverter` doesn't have to be further nested in a `wcsapi.wrappers.sliced_wcs` submodule).

Reply:
Agreed, moving them to a `wcs` submodule makes sense and will keep things organized.